README.Rmd

```diff
@@ -47,133 +49,21 @@ To load the package, use
-these simulations. 
-
-For example, if the probability of observing each case is `obs_prob = 0.30`, 
-we use
-
-```{r}
-chain_sizes <- c(1, 1, 4, 7) # example of observed chain sizes
-ll <- chain_ll(chain_sizes, "pois", "size", obs_prob = 0.3, lambda = 0.5,
-               nsim_obs = 10)
-ll
-```
-
-This returns `10` likelihood values (because `nsim_obs = 10`), which can be 
-averaged to come up with an overall likelihood estimate.
-
-To find out about usage of the `chain_ll()` function, you can use the `R` help 
-file
-
-```{r eval=FALSE}
-?chain_ll
-```
+* `chain_ll()`: calculates the likelihoods of observing a vector of chains 
+  of given sizes or lengths.
 
-### How `chain_ll()` works
+* `chain_sim()`: simulates transmission chains until all chains die out.
 
-If the probability distribution of chain sizes or lengths has an analytical 
-solution, this will be used. `chain_ll()` currently supports the Poisson and 
-negative binomial size distribution and the Poisson and geometric length 
-distribution. 
+* `chain_sim_susc()`: simulates transmission chains until a specified initial 
+  susceptible pool runs out.
 
-If an analytical solution does not exist, simulations are used to approximate 
-this probability distributions ([using a linear approximation to the cumulative 
-distribution](https://en.wikipedia.org/wiki/Empirical_distribution_function) 
-for unobserved sizes/lengths). In that case, an extra argument `nsim_offspring` 
-must be passed to `chain_ll()` to specify the number of simulations to be 
-used for this approximation. 
 
-For example, to get offspring drawn from a binomial distribution with 
-probability `prob = 0.5`, we run
-
-```{r}
-chain_sizes <- c(1, 1, 4, 7) # example of observed chain sizes
-chain_ll(chain_sizes, "binom", "size", size = 1, prob = 0.5,
-         nsim_offspring = 100)
-```
-
-## Simulating branching processes
-
-To simulate a branching process, we use the `chain_sim()` function. This 
-function follows the same syntax as `chain_ll()`.
-
-Below, we are simulating $5$ chains, assuming the offspring are generated using
-a Poisson distribution with mean, `lambda = 0.5`. By default, `chain_sim()` 
-returns a vector of chain sizes/lengths. If we instead want to return 
-a tree of infectees and infectors, we need to specify a function for 
-the serial interval and set `tree = TRUE` (see next section).
-
-```{r}
-chain_sim(n = 5, offspring = "pois", stat = "size", lambda = 0.5)
-```
-
-### Simulating trees
-
-To simulate a tree of transmission chains, we specify the serial interval 
-generation function and set `tree = TRUE` as follows:
-
-```{r}
-set.seed(13)
-serial_interval <- function(n) {
-  rlnorm(n, meanlog = 0.58, sdlog = 1.58)
-}
-chains_df <- chain_sim(
-  n = 5, offspring = "pois", lambda = 0.5, stat = "length",
-  infinite = 100, serial = serial_interval, tree = TRUE
-)
-head(chains_df)
-```
+See the ["introduction vignette"](bpmodels) for a detailed illustration of 
```

Comment:
Looks good to me. It communicates directly the core actions of the package's main functions and gives more space to the "Get started" vignette. 
Some details: 
- this link is not working for me locally. Probably `(articles/bpmodels)` works? I'm rendering it with `pkgdown::build_site()`
- Replacing `"introduction vignette"` with `"Get started" vignette` could indicate clearly to the content in the navbar called "Get started". Possibly too obvious, but it makes me think if there is another intro vignette different to the get started.

With these comments, I can close my review. I'll give further feedback on the "Get started" vignette content in separate issues.

Reply:
Thanks, Andree. I have fixed that in 23f44e8b69f7b4d9d672e1e21707ef287797b1ca.